src/meta/src/hummock/manager/time_travel.rs

```diff
@@ -342,7 +346,7 @@ impl HummockManager {
         let sst_count = sst_ids.len();
         let mut sst_id_to_info = HashMap::with_capacity(sst_count);
         let sst_info_fetch_batch_size = std::env::var("RW_TIME_TRAVEL_SST_INFO_FETCH_BATCH_SIZE")
-            .unwrap_or_else(|_| "100".into())
+            .unwrap_or_else(|_| "10000".into())
```

Comment:
Shall we move this env var to the meta opt instead of implicitly reading an env var here?

And any reason to change from 100 to 10000?

Reply:
> Shall we move this env var to the meta opt instead of implicitly reading an env var here?

OK

> And any reason to change from 100 to 10000?

To speed up time travel version replay, i.e. less SELECT queries.